src/Database/PostgreSQL/PQTypes/Model/Index.hs

```diff
@@ -154,9 +154,9 @@ sqlCreateIndex_ concurrently tname idx@TableIndex{..} = mconcat [
   , if idxUnique then " UNIQUE" else ""
   , " INDEX "
   , if concurrently then "CONCURRENTLY " else ""
-  , indexName tname idx
+  , "\"", indexName tname idx, "\""
```

Comment:
Turns out that you'll have quote every occurence of `indexName` that makes it's way into SQL query.
I suggest to make quoting function like `dquoted`.

Same change might be also appropriate in `PrimaryKey.hs` and `ForeignKey.hs`.

Reply:
Done.